keyboards/handwired/misterdeck/misterdeck.c

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2021 Chris Broekema (broekema@gmail.com)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "misterdeck.h"
+
+void matrix_init_kb(void) {
+  matrix_init_user();
+}
```

Comment:
```suggestion
```

Reply:
done in commit https://github.com/qmk/qmk_firmware/pull/15556/commits/cc306d29f6926acd96ada0014321efabb7eeb7be